src/main/java/appeng/fluids/parts/PartFluidExportBus.java

```diff
@@ -118,17 +113,12 @@ protected TickRateModulation doBusWork()
 
 				if( fh != null )
 				{
-					for( int i = 0; i < this.getConfig().getSlots(); i++ )
+					for( int i = 0; i < this.getConfig().length; i++ )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)